iModelCore/ECDb/PublicAPI/ECDb/IECSqlBinder.h

```diff
@@ -10,6 +10,40 @@
 
 BEGIN_BENTLEY_SQLITE_EC_NAMESPACE
 
+//=======================================================================================
+// @bsiclass
+//+===============+===============+===============+===============+===============+======
+struct BinderInfo final

```

Comment:
I've thought about this one for a while. I understand you want to encapsulate the two pieces of information here.
But overall, it seems we'd be better off just putting this into ECSqlBinder directly, or is there some reason not to?
Requires less code and maintenance, and BinderInfo is not a very good term, it doesn't provide context on what info is encapsulated, while other info sits on the Binder directly.
I'd suggest to get rid of this struct.